Packs/Threat_Crowd/Integrations/ThreatCrowdV2/ThreatCrowd_v2_description.md

```diff
@@ -0,0 +1,4 @@
+## BaseIntegration Help
+
+ThreatCrowd is a system for finding and researching artefacts relating to cyber threats.
+This integration allows querying ThreatCrowd for reports on common indicators types.
```

Comment:
this is usually for instance config help. can be rmed.
the desc though(without the typos), is more fitting for the yml and pack description.